app/assets/styles/styles.css

```diff
@@ -33,194 +31,120 @@ p {
   line-height: 1.55;
 }
 
-.newsletter-date {
-  color: #fdfd56;
-  font-size: 0.8rem;
+.logo__btn {
+  display: flex;
+  justify-content: center;
+  margin-top: 0.7em;
+  margin-bottom: 0.7em;
 }
 
-.language-menu {
-  width: 100%;
-  font-size: 1.15rem;
-  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
-  text-align: right;
-  background: #1a1451;
-  margin-bottom: 0.625em;
-  padding: 0.5em 1em;
+@media only screen and (min-width: 400px) {
+  .logo__btn {
+    display: block;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    margin-left: 1.5em;
```

Comment:
When positioning an element with `absolute` I like to use the `top`, `left`, `right`, `bottom` properties to control the positioning of the element because it ties the concepts together. You can use margin, there's nothing incorrect. Again it's more about using like sets of properties. 

When you make the change to the menu you'll be able to remove the `margin-top` and `margin-left` properties. Please add `top: 0.5em` so the logo has a bit of breathing room. It makes the design pop a bit more to see some of the blue nav above the logo. It also showcases the full shield and gives it a bit more prominence.